core/contracts/contracts/privacy/lib/MerkleTree_SHA256.sol

```diff
@@ -0,0 +1,335 @@
+// SPDX-License-Identifier: CC0
+
+/**
+ * A base contract which handles Merkle Tree inserts (and consequent updates to the root and 'frontier' (see below)).
+ * The intention is for other 'derived' contracts to import this contract, and for those derived contracts to manage 
+ * permissions to actually call the insertLeaf/insertleaves functions of this base contract.
+ * 
+ * credit: https://github.com/EYBlockchain/timber
+**/
+
+pragma solidity ^0.6.9;
+
+
+contract MerkleTree_sha256 {
```

Comment:
This should follow the Solidity style guide for naming... so `MerkleTreeSHA256` and `MerkleTreeSHA256.sol`.

See https://solidity.readthedocs.io/en/v0.5.3/style-guide.html#contract-and-library-names

Reply:
Fixed